api-reference/documents/update.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: "Update Document"
+api: "POST https://api.getmetal.io/v1/index"
+description: "This endpoint updates and replaces an existing Document(embedding) with the inputted data."
+---
+
+### Auth Headers
+
+<ParamField required header="x-metal-api-key" type="string">
+  An API key for your org.
+</ParamField>
+
+<ParamField required header="x-metal-client-id" type="string">
+  A Client ID for your org.
+</ParamField>
+
+### Body
+
+<ParamField body="index" type="string" placeholder="Index ID" required>
+  Index ID
+</ParamField>
+
+<ParamField body="id" type="string">
+  Identifier for the embedding you wish to replace. Must be between 3 - 256 chars and consist of
+  numbers, letters, or `-`.
+</ParamField>
```

Comment:
the `id` should be more obvious i think -- this would be necessary in order to run an "update"